test/guessing-game-test.js

```diff
@@ -0,0 +1,13 @@
+const GuessingGame = require('../lib/guessing-game');
+const assert = require('chai').assert;
+
+describe("GuessingGame", function(){
+  it("allows user to guess number for object", function(){
```

Comment:
ideally, we'd have separate tests for each function we have available on your `GuessingGame` object, but since this object is relatively simple, this is probably fine!